### DISCUSSION TITLE
Fleet 1.32 not supported yet

### DISCUSSION THREAD

**Discussion by danny-su** [izhangzhihao/intellij-rainbow-brackets#2713] [Q&A] [answered]
找不到marketplace里面的插件。
<img width="565" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/ce853c93-b6ed-446d-9bbb-f99297ff3e4f">
<img width="500" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/6f2ff237-859c-49d9-ba5b-71ca180133f1">

**Comment by danny-su** [comment]
@izhangzhihao

**Comment by izhangzhihao** [comment]
<img width="882" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/fb2e8f27-fbf5-443e-aa2c-fbbd354cebef">

**Comment by danny-su** [comment]
@izhangzhihao 你是什么版本的fleet，为啥我找不到呢，我的fleet版本：
```
Fleet version: build 1.32.118
OS: Mac OS X (14.4, aarch64)
```
我的fleet上就不能安装第三方插件，这个是需要在fleet里面打开什么设置吗？

**Comment by izhangzhihao** [comment]
Fleet 1.32 not supported yet, I will try created a new release ASAP.

**Comment by danny-su** [comment]
@izhangzhihao 你的fleet是啥版本，我这个已经是通过jetbrains toolbox安装的最新的fleet了。。。

**Comment by izhangzhihao** [comment]
1.31是支持的。

**Reply by danny-su** [reply]
等你最新版，刚买了一个永久license，麻烦折扣码发我邮件，谢谢！

**Reply by izhangzhihao** [reply]
邮件发了，请查收呢，没收到的话看一下垃圾收件箱

**Comment by izhangzhihao** [comment, marked as answer]
It's available now. Pls try download again.

**Reply by danny-su** [reply]
怎么安装呢，插件查找里面还是找不到啊。
<img width="187" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/e7949c7f-eaae-4198-92d1-bf40c2e309dc">

**Reply by izhangzhihao** [reply]
> 怎么安装呢，插件查找里面还是找不到啊。
> 
> <img width="187" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/e7949c7f-eaae-4198-92d1-bf40c2e309dc">
> 
> 

就是在这里安装， 搜不到你试着重启一下？ 是不是网络有问题之类的

**Reply by danny-su** [reply]
@izhangzhihao 这个肯定不是网络问题，我都重装过了，我这里就是看不到marketplace里面的东西，难道这玩意有什么地方开关吗？

**Reply by izhangzhihao** [reply]
我自己试了是可以搜到并下载的， marketplace 上也确实有https://plugins.jetbrains.com/plugin/23759-rainbow-brackets-for-fleet/

**Reply by danny-su** [reply]
@izhangzhihao 我这边就是死活找不到，你看到我的图和你的图的区别，我这边没有看到maketplace，只有installed。

<img width="323" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/7aace899-7f13-42a5-afa6-2af4a6f9b612">

![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12178855/692b32bd-66d7-44cc-83ba-ca629da09bd7)

**Reply by izhangzhihao** [reply]
试试这个

Enable internal mode for Fleet: add "internalMode": true, into Fleet settings.json file

<img width="582" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/fa5f9928-aa92-4a49-8478-c6e8a46691ee">


<img width="368" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/4e3037b8-7e3e-4854-b2ac-10c212018e80">

**Reply by danny-su** [reply]
@izhangzhihao 果然有效，谢谢:)
